### PR TITLE
feat : profile page

### PR DESCRIPTION
**Pull request by rohit8686**
Features : 

- User profile
- Address
- Order details

### CODE REVIEW

File: src/frontend/Routes/Routing.jsx [hunk 1]
```diff
@@ -25,6 +29,11 @@ export const Routing = () => {
       <Route path="/products/:productId" element={<Product />} />
       <Route path="/checkout" element={<Checkout />} />
       <Route path="/address" element={<Address />} />
+      <Route path="/profile" element={<Profile />}>
+        <Route index element={<UserProfile />} />
+        <Route path="addressDetails" element={<AddressDetails />} />
```
Comment (Parul1999) [inline src/frontend/Routes/Routing.jsx:34]: / is missing in both the pathnames

File: src/frontend/pages/Profile/Profile.jsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+import { NavLink, Outlet } from "react-router-dom";
+
+export const Profile = () => {
+  return (
+    <div>
+      <div className="flex border">
+        <NavLink to="/profile" className="link" end>
+          User Profile
+        </NavLink>
+        <NavLink to="addressDetails" className="link">
```
Comment (Parul1999) [inline src/frontend/pages/Profile/Profile.jsx:12]: / is missing in to for address and order details

File: src/frontend/pages/Profile/Profile.jsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+          Address
+        </NavLink>
+        <NavLink to="orderDetails" className="link">
+          Order Details
+        </NavLink>
+      </div>
+      <div className="flex">
```
Comment (Parul1999) [inline src/frontend/pages/Profile/Profile.jsx:19]: Try avoiding giving classnames which are properties/reserved words.

File: src/frontend/Routes/Routing.jsx [hunk 4]
```diff
@@ -11,6 +11,10 @@ import { PageNotFound } from "../pages/PageNotFound/PageNotFound";
 import { Product } from "../pages/Product/Product";
 import { Checkout } from "../pages/Checkout/Checkout";
 import { Address } from "../pages/Address/Address";
+import { Profile } from "../pages/Profile/Profile";
```
Comment (Parul1999) [inline src/frontend/Routes/Routing.jsx:14]: Suggestion -The imports from same component ,  e,g here from pages can be put up in one file and import that single file

File: src/frontend/Routes/Routing.jsx (hunk 1 above)
Comment (rohit8686) [reply src/frontend/Routes/Routing.jsx:34]: for nested routes we do not need to give /

File: src/frontend/pages/Profile/Profile.jsx (hunk 2 above)
Comment (rohit8686) [reply src/frontend/pages/Profile/Profile.jsx:12]: same here we do not need to give /

**Review by RahulSingh9131 — APPROVED** [review]
clean and understandable code, go ahead and merge.

**Pull request merged**